README.md

```diff
@@ -19,6 +11,15 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
 #############################################################################
 ```
+---
+Upcomming changes:
+- Crucible's roles have been migrated to the [Red Hat CI OCP Ansible Collection](https://github.com/redhatci/ansible-collection-redhatci-ocp) available for download at [Ansible Galaxy](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/).  
+- Crucible's playbooks will continue to work but as always, you are encouraged to fork crucible and maintain your own playbooks.
+- On <ins>Monday April 18th</ins> Crucible will crossover and default to using the *redhatci.ocp collection*. At this point all roles will be removed from `roles/`.  
```

Comment:
most roles (there are two that we will be putting back)

Reply:
Validate and what else is staying? You mentioned it earlier but I couldn't pay attention.

Reply:
The display plan